sql/6_0_0-to-6_1_0_upgrade.sql

```diff
@@ -867,4 +867,35 @@ CREATE INDEX `abook_type` ON `users` (`abook_type`);
 
 #IfNotIndex procedure_type ptype_procedure_code
 ALTER TABLE `procedure_type` ADD INDEX `ptype_procedure_code`(`procedure_code`);
+#EndIf
+
+#IfNotTable form_vital_details
+CREATE TABLE `form_vital_details` (
+  `id` 	bigint(20) NOT NULL AUTO_INCREMENT,
+  `form_id` bigint(20) NOT NULL COMMENT 'FK to vital_forms.id',
+  `vitals_column` varchar(64) NOT NULL COMMENT 'Column name from form_vitals',
+  `interpretation_list_id` varchar(100) DEFAULT NULL COMMENT 'FK to list_options.list_id for observation_interpretation',
+  `interpretation_option_id` varchar(100) DEFAULT NULL COMMENT 'FK to list_options.option_id for observation_interpretation',
+  `interpretation_codes` varchar(255) DEFAULT NULL COMMENT 'Archived original codes value from list_options observation_interpretation',
+  `interpretation_title` varchar(255) DEFAULT NULL COMMENT 'Archived original title value from list_options observation_interpretation',
+  PRIMARY KEY (`id`),
+  KEY `fk_list_options_id` (`interpretation_list_id`,`interpretation_option_id`),
+  CONSTRAINT `fk_form_id` FOREIGN KEY (`form_id`)
```

Comment:
![2DV](https://user-images.githubusercontent.com/278968/126410510-99a08f16-8030-4818-9cd8-37e0f16eba4f.gif)

Reply:
Glad you like it haha.